src/modules/Makefile.am

```diff
@@ -287,6 +287,25 @@ CLEANFILES += $(EXTRA_sd_kali_DEPENDENCIES)
 endif
 endif
 
+#
+#  cxxpiper
+#
+modulebin_PROGRAMS += sd_cxxpiper
+PIPER_LIB_DIR = /usr/lib/piper
+PIPER_SRC_DIR = /usr/local/src/piper
+sd_cxxpiper_SOURCES = cxxpiper.cpp module_utils_addvoice.c module_utils_play.c $(common_SOURCES)
+sd_cxxpiper_CPPFLAGS = -I/usr/include/piper_phonemize/include/ \
+	-I$(PIPER_SRC_DIR)/src/cpp/ \
+	$(AM_CPPFLAGS)
+sd_cxxpiper_LDADD = $(top_builddir)/src/common/libcommon.la \
+	-L$(PIPER_LIB_DIR) \
+	-lpiper_phonemize \
+	-lonnxruntime \
+	-lespeak-ng \
+	-lrubberband \
+	$(SNDFILE_LIBS) \
+	$(common_LDADD)
```

Comment:
We really need to make this optional, because at lot of people will not have piper installed on their system. Actually pipe don't seem to have a real installation makefile target that would properly install header files? This really should be fixed by upstream, otherwise it'll be a mess...

This is normally done in `configure.ac`, you can see examples, see for instance `check for svox pico support` which checks for the presence of the `libttspico` library before enabling the `pico_support` condition, which is then used in `Makefile.am`.